### PR TITLE
fix(i18n): supplement missing Chinese translation fields

### PR DESCRIPTION
**Pull request by WangYJEE**
…translation

Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by tolking** [comment]
It seems that there have been some conflicts. Can you resolve them first?

**Review by tolking — APPROVED** [review]

**Review by btea — APPROVED** [review]
Thanks!

File: packages/locale/lang/zh-cn.ts [hunk 1]
```diff
@@ -94,18 +114,30 @@ export default {
...
     upload: {
       deleteTip: '按 delete 键可删除',
       delete: '删除',
       preview: '查看图片',
       continue: '继续上传',
     },
+    slider: {
+      defaultLabel: '滑块介于 {min} 至 {max}',
+      defaultRangeStartLabel: '选择起始值',
+      defaultRangeEndLabel: '选择結束值',
```
Comment (warmthsea) [inline packages/locale/lang/zh-cn.ts:139]: `結`  `结`
这里错了，用成繁体了

Comment (btea) [reply packages/locale/lang/zh-cn.ts:139]: 谢谢指出问题,，欢迎PR。

Comment (tolking) [reply packages/locale/lang/zh-cn.ts:139]: 确实，审核失误，看了需要另外提一个PR修复它

Comment (btea) [reply packages/locale/lang/zh-cn.ts:139]: 我觉得最好分开，这里是修复中文翻译。你提供的那个PR是新增其他语言的翻译，可能会审核比较慢。

**Pull request merged**